Mobile/starter_project_dartsmiths/lib/core/errors/failures.dart

```diff
@@ -2,6 +2,6 @@ import 'package:equatable/equatable.dart';
 
 abstract class Failure extends Equatable {
   const Failure({List properties = const <dynamic>[]});
-  
-  
 }
+
+class ServerFailure implements Exception {}
```

Comment:
the server failure should be of type failure not of type exception. Write another class that will implement exceptions if you want to implement a specific one.